ScoutSuite/providers/aws/resources/sqs/service.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from ScoutSuite.providers.aws.resources.regions import Regions
+from ScoutSuite.providers.aws.resources.resources import AWSResources
+
+
+class RegionalQueues(AWSResources):
+    async def fetch_all(self, **kwargs):
+        queue_urls = await self.facade.sqs.get_queues(self.scope['region'])
+        # TODO: parallelize this async loop:
+        for queue_url in queue_urls:
+            id, queue = await self._parse_queue(queue_url)
+            self[id] = queue
+
+    async def _parse_queue(self, queue_url):
+        queue = {'QueueUrl': queue_url}
+        attributes = await self.facade.sqs.get_queue_attributes(
+            self.scope['region'], queue_url, ['CreatedTimestamp', 'Policy', 'QueueArn', 'KmsMasterKeyId']
+        )
```

Comment:
In my other PRs, I usually tried to put all the API calls in the facade call. This way, all the data is retrieved at the same place (in the facade), and the parse method really only does parsing. I think it's a good convention to follow, it respects the separation of concerns principle.

Reply:
You're perfectly right (and to be honest, I wasn't really satisfied with that)! I just fixed the separation of concerns. :)